### DISCUSSION TITLE
Day 1 - Introduction

### DISCUSSION THREAD

**Discussion by OndrejMottl** [CUNI-SPRouT/SPROuT-Forum#3] [Course workspace]
Work materials created during SPROuT 2024 - Day 1

**Comment by OndrejMottl** [comment]
Open Science Mind maps - comment here!

**Reply by OndrejMottl** [reply]
A comment

**Reply by SvandaPetr** [reply]
![mindmap](https://github.com/user-attachments/assets/ac2863e2-afdc-42e8-99d5-4c0e13a51cd5)

**Reply by HelenaPijalkova** [reply]
![open science](https://github.com/user-attachments/assets/9870805e-8b4e-44f1-9723-eaa9f2fa4727)

**Reply by FernandoPGaona** [reply]
Group: Fernando Gaona & Fotoula Papandreou
![OpenScience](https://github.com/user-attachments/assets/eb9c34e1-a23e-43e2-8a4a-b8fccb7dd2bf)

**Reply by BiAlpha** [reply]
<img width="1138" alt="Screenshot 2024-11-05 at 3 18 37 PM" src="https://github.com/user-attachments/assets/2cdf5d58-ff7a-4771-8bc7-92a0b69c0ba1">

**Reply by KadyanSonia** [reply]
> Open Science Mind maps - comment here!
![MindMap_Open Science](https://github.com/user-attachments/assets/adc19dde-1536-4887-99d3-60328ff839b9)

**Reply by ChonourMahmoudi** [reply]
> Open Science Mind maps - comment here!

![Uploading Image 2024-.jpg…]()

**Comment by OndrejMottl** [comment]
How to improve Reproducible Research?
